api/system-authorization/read

```diff
@@ -96,6 +98,6 @@ print encode_json({
     "status" => {
         "isRoot" => $isRoot,
         "isAdmin" => $isAdmin,
-        "username" => scalar getpwuid $<
+        "username" => $username
```

Comment:
This was present also on the previous PR.
Where is the new `username` parameter needed?

Reply:
above

Reply:
The `username` paramater is passed to the UI. What is for?
Do you just need to access the current logged user name from the UI?

Reply:
the user's name is needed to find the path of the user's home